etc/openstack-config/openstack-config.yml

```diff
@@ -96,6 +96,48 @@
 # Whether generated cluster templates should be public by default (defaults to 'True')
 # magnum_cluster_templates_public:
 
+###############################################################################
+# Configuration variables for a CloudKitty ratings service deployment.
+
+#     #    ###   #######  #####    IN ORDER TO SUCCESSFULLY DEPLOY CLOUDKITTY
+# #   #   #   #     #     #        'cloudkitty.conf' & 'metrics.yml' MUST BE    
+#  #  #   #   #     #     ###      PROVIDED ALONGSIDE THE FOLLOWING CONFIG.
+#   # #   #   #     #     #        MORE INFO CAN BE FOUND IN THE README.md
+#     #    ###      #     #####    'stackhpc-kayobe-config' DOCS.
+
+# File path or URL containing Python upper constraints.
+####### THIS IS NOT NECCESSARY TO CONFIGURE ###########
+#openstack_ratings_upper_constraints_file: /path/to/upper_constraints_file
+
+# Environment variables for the OpenStack CLI. By default this is left empty.
+################### THIS IS NOT NECCESSARY TO CONFIGURE #####################
+#openstack_ratings_environment:
+#  - name: ENV_VAR_NAME
+#    value: ENV_VAR_VALUE
+
+# A list where each item is a dictionary mapping the associated fields, 
+# with the 'mappings' field also being a list of dictionaries. 
+# Example of the mappings and their fields can be found below, 
+# however for more information please refer to the README.md file.
+#
+#openstack_ratings_hashmap_field_mappings:
+#  - service: SERVICE_NAME
+#    name: FIELD_NAME
+#    mappings:
+#      - value: MAPPING_VALUE
+#        cost: MAPPING_COST
+#        group: MAPPING_GROUP
```

Comment:
Mark as optional?

Reply:
I thought it was only the `group` which was optional, however GitHub showed the comment as highlighting and reffering to the entire `mapping` so I thought I simply misunderstood.